html/styles/style.css

```
@@ -0,0 +1,356 @@
+.page {
+  max-width: 1440px;
+  min-width: 950px;
+  background-color: #fff;
+  font-family: 'Abel', sans-serif;
+}
+
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  position: relative;
+  min-height: 89px;
+  background: #fff;
+}
+
+.logo {
+  position: absolute;
+  width: 55px;
+  height: 55px;
+  border-radius: 50%;
+}
+
+.logo_place_header {
+  left: 50px;
+  top: 17px;
+  bottom: 17px;
+  background: #D8D8D8;
+}
+
+.navigation {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__navigation {
+  margin-right: 50px;
+  margin-top: 29px;
+  margin-bottom: 29px;
+}
+
+.rectangle-link {
+  width: 154px;
+  height: 31px;
+  margin-right: 40px;
+}
+
+.rectangle-link:last-of-type {
+  margin-right: 0;
+}
+
+.header__link {
+  background: #D8D8D8;
+}
+
+.link:last-of-type {
+  margin-right: 0;
+}
+
+.banner {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  min-height: 450px;
+  background-color: #d8d8d8;
+}
+
+.banner__title {
+  width: 330px;
+  height: 76px;
+  margin-top: 120px;
+  margin-left: 130px;
+  margin-bottom: 0;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 60px;
+  line-height: 76px;
+}
+
+.banner__subtitle {
+  width: 375px;
+  height: 46px;
+  margin-left: 130px;
+  margin-top: 1px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 36px;
+  line-height: 46px;
+}
+
+.banner__buttons {
+  display: flex;
+  justify-content: space-between;
+  width: 591px;
+  margin-top: 29px;
+  margin-left: 130px;
+}
+
+.button {
+  width: 283px;
+  height: 60px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 26px;
+  line-height: 33px;
+}
```

```suggestion
.button {
  width: 283px;
  height: 60px;
  font-style: normal;
  font-weight: normal;
  font-size: 26px;
  line-height: 33px;
  
   border: none;
}
```
необходимо убрать границы у кнопок. В chrome отображается некорректно